### PR TITLE
Add another way to change the commit message.

### PR DESCRIPTION
**Pull request by appleboy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,13 @@ If you wrote the wrong thing and the commit has not yet been pushed, you can do
 $ git commit --amend
 ```
 
+The another way to change commit message:
```
Comment (unixorn) [inline README.md]: I'd rephrase this as
`Another way to change the commit message:`

Comment (RichardLitt) [inline README.md]: Agreed. Besides that, all good!

Comment (appleboy) [inline README.md]: Done!

**Comment by RichardLitt** [comment]
:+1:
